src/main/java/fr/insalyon/citi/golo/compiler/ParseTreeToGoloIrVisitor.java

```diff
@@ -35,6 +36,10 @@
 
   private GoloCompilationException.Builder exceptionBuilder;
 
+  private static String syntheticName(String base) {
+    return "__$$_" + base + "_" + System.currentTimeMillis();
```

Comment:
As I said, I already made a alternative implementation in the macro branch (see https://github.com/yloiseau/golo-lang/blob/wip/macro-quote/src/main/java/gololang/macros/SymbolGenerator.java). However, since I don't know when it will be ready, I can already make change (simply unsing a counter) in this PR.